configurations/docker_image_name.txt

```diff
@@ -1 +1 @@
-ghcr.io/africasvoices/engagement-data-pipeline:handle-whatsapp-urns-rapid-pro-to-engagement-db-1
+ghcr.io/africasvoices/engagement-data-pipeline:handle-whatsapp-urns-rapid-pro-to-engagement-db-2
```

Comment:
What's the status of a PR and deployment of this branch? I think using WhatsApp might need some special care in the pipelines so we need to be careful not to use it before it's ready.

Reply:
Its ready for deployment. Discussed offline with DT.

Reply:
In that case can you release and use the released version?